site/src/examples/pagination/CompactWithInput.tsx

```diff
@@ -0,0 +1,8 @@
+import { ReactElement } from "react";
+import { Pagination, CompactPaginator } from "@salt-ds/lab";
+
+export const CompactWithInput = (): ReactElement => (
+  <Pagination count={20}>
+    <CompactPaginator withInput />
```

Comment:
why are we using a prop `withInput` rather than composing?

```
<CompactPaginator>
    <PagaintorInput />
</CompactPaginator>
```

Reply:
I found it easier this way